Gemfile

```diff
@@ -2,7 +2,7 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gem "logstash-core", "2.0.0.dev", :path => "."
+gem "logstash-core", "2.0.0.snapshot1"
```

Comment:
why not using here something like " ~> 2.0.0.snapshot" then all snapshot + new non pre release version will be into this pattern? I think makes sense. What do you think?